### PR TITLE
ref#5 create section introduction consultans

### PR DESCRIPTION
**Pull request by ttmthanh**
Result:
1. PC
image
2. Mobile
image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -819,6 +819,35 @@ <h3 class="font-medium pt-20 pb-8 text-[20px] md:text-[32px] text-black">登録
...
+    <div id="jobs" class="bg-white text-center pb-12 md:pb-auto">
+      <h3 class="font-semibold pt-20 pb-16 text-[20px] md:text-[32px]">コンサルタントにご相談ください</h3>
+      <div class="grid">
+        <div class="md:flex md:gap-4 p-3 text-[16px] justify-center mx-auto">
+          <div class="mb-4 border-solid border-2 border-black shadow-md md:w-1/3 bg-white rounded-lg border shadow-md shadow-4px hover:shadow-2px hover:bg-gray-100 dark:border-gray-700 dark:bg-gray-800 dark:hover:bg-gray-700">
+            <a href="https://www.facebook.com/profile.php?id=100051502774248">
+              <img class="rounded-t-lg w-full mb-4" src="src/images/uyen.png" alt="avatar">
+              <span class="font-medium">Phan Thi Phuong Uyen</span>
+              <div class="p-5 text-[12px] md:text-[16px] text-center">
+                <button href="https://www.facebook.com/profile.php?id=100051502774248" class="text-[14px] md:text-[18px] transition-all ease-linear delay-0 hover:-translate-y-1 hover:scale-110 duration-500 font-medium btn bg-red-500 text-center border-solid border-2 mr-4 border-black rounded-xl px-4 py-2 shadow-4px hover:shadow-2px hover:brightness-95 text-white">
```
Comment (yosiyuki) [inline index.html]: You can't set href in `<a href >`

**Review by yosiyuki — CHANGES_REQUESTED** [review]
1. Please make image smaller as view size
2. We don't need 2 uyen pics

File: index.html [hunk 2]
```diff
@@ -819,6 +819,35 @@ <h3 class="font-medium pt-20 pb-8 text-[20px] md:text-[32px] text-black">登録
...
+                </div>
+              </div>
+            </a> 
+          </div>
+        </div>
+      </div>
+    </div>
     <footer class="h-[10.5rem]">
       <div class="bg-blue text-center py-8 text-[10px] md:text-[14px]">
         <span class="text-white px-4">Copyright © 2020 KOSAIDO BUSINESS SUPPORT CO., LTD. All Rights Reserved</span>
```
Comment (yosiyuki) [inline index.html]: Please update 2020 -> 2022

Comment (ttmthanh) [reply index.html]: I updated again

**Review by yosiyuki — APPROVED** [review]

**Pull request merged**